docs/logging/usage.rst

```diff
@@ -298,12 +314,29 @@ All logs will go to a single custom log, which defaults to "python". The name
 of the Python logger will be included in the structured log entry under the
 "python_logger" field. You can change it by providing a name to the handler:
 
-.. literalinclude:: logging_snippets.py
+.. literalinclude:: snippets.py
     :start-after: [START create_named_handler]
     :end-before: [END create_named_handler]
 
+Cloud Logging Handler transports
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The :class:`~google.cloud.logging.handlers.handlers.CloudLoggingHandler`
+logging handler can use different transports. The default is
+:class:`~google.cloud.logging.handlers.BackgroundThreadTransport`.
+
+ 1. :class:`~google.cloud.logging.handlers.BackgroundThreadTransport` this is
+    the default. It writes entries on a background
+    :class:`python.threading.Thread`.
+
+ 1. :class:`~google.cloud.logging.handlers.SyncTransport` this handler does a
```

Comment:
[It doesn't](http://temp.theadora.io/google-cloud-python/logging/usage.html#cloud-logging-handler-transports), but not something I'm seeking to fix in this PR.